Comp2023/src/main/java/frc/robot/commands/ElbowMoveToAngle.java

```diff
@@ -0,0 +1,56 @@
+
+// ROBOTBUILDER TYPE: Command.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants.ARMConsts.ElbowAngle;

```

Comment:
Does this mean that elbow and wrist constants are coming from ARMConsts? Shouldn't these be in separate constants (ie ELConsts and WRConsts)?